### DISCUSSION TITLE
Nextjs and AutoCancellation

### DISCUSSION THREAD

**Discussion by TheHypnoo** [pocketbase/pocketbase#5591] [Q&A]
Hi guys, I'm creating a simple project. Where mainly the subscription section of pocketbase is used.

In this case I have a simple example, I have to get the data and subscribe to it, from two different places in the same page. Example: Sidebar (component) and the content of the page.

This causes the AutoCancellation error. What would be the way to solve this?

Mainly I have the instance for the middleware with the Auth and then I have a general instance type Singleton which is for the client side. 

Any help is welcome.

Thanks in advance.

**Comment by orenaksakal** [comment]
I'm not sure what is the problem exactly, would be nice to see some code and I think it might be related to admin sdk if thats the case
you can use `pb.autoCancellation(false)` to "Globally enable or disable auto cancellation for pending duplicated requests." (based on the error you are seeing)

**Reply by TheHypnoo** [reply]
This is more of a Nextjs problem than a Pocketbase problem. In this case with ReactQuery, I solved it.
